source/_lectures/getting-started-with-sylius.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "Getting started with Sylius"
+
+# číslo uživatele v users.yml
+user: 118
+# @todo: mentors - doplnit?
+
+# obvykle půldenní 3Vladimír Macháček-4 hodiny nebo celodenní 8 (včetně oběda)
+duration: "8 hodin"
+
+# x999, obvykle 3 k za půldenní, 5 k za celodenní
+price: 4999
+
+# max. počet lidí, který zvládneš a je ještě efektinví pro předávání tvého know-how
+# doporučujeme: 6-10
+capacity: "10 lidí"
+
+image: "/assets/images/logos/sylius.png"
+
+perex: "Comprehensive introduction to Sylius E-Commerce framework. Join us and learn how to customize one of the most promising e-commerce solution."
+---
+
+Have you heard about Sylius E-Commerce framework? If so, you know already that it is an important part of Symfony ecosystem and it is one of the biggest, publicly available projects developed in BDD methodology. If not, join me and check that e-commerce development can be a pleasure.
+
+<br>
+
+### Co si ze školení odneseš?
+
+- Being familiar with Sylius architecture and bundles separation.
+- Creating a CRUD API in minutes.
+- Creating a new resource with Sylius Resource Bundle.
+- How to smoothly generate Admin CRUD scaffolding with Sylius Grid Bundle.
+- Learning how to use State Machine component in Sylius
+- How to improve project development with fixtures generation
+- Sending emails dispatched by Sylius Resource Events
+
+<br>
+
+### Co je potřeba před školením
+
+- Installed PHP 7.1
+- Installed [PHPStorm](https://www.jetbrains.com/phpstorm/download/)
+- Setted up [Sylius-Standard edition](http://docs.sylius.com/en/latest/book/installation/index.html)
+- Basic Symfony knowladge
+
+<br>
+
+After training you should be familiar with Sylius eco system and you should be able to decide if Sylius suites your needs.
```

Comment:
with Sylius ecosystem -> with Sylius basics and will be able to start your first e-commerce project.